code/controllers/Processes/lighting.dm

```diff
@@ -1,62 +1,60 @@
+// Solves problems with lighting updates lagging shit
+// Max constraints on number of updates per doWork():
+#define MAX_LIGHT_UPDATES_PER_WORK		100
+#define MAX_OVERLAY_UPDATES_PER_WORK	1000
+
 /datum/controller/process/lighting
 	schedule_interval = LIGHTING_INTERVAL
 
-	// Solves problems with lighting updates lagging shit
-	// Max constraints on number of updates per doWork():
-	var/const/MAX_LIGHT_UPDATES_PER_WORK=100
-	var/const/MAX_OVERLAY_UPDATES_PER_WORK=1000 // idfk
-
-	// Counters
-	var/light_updates=0
-	var/overlay_updates=0
-
 /datum/controller/process/lighting/setup()
 	name = "lighting"
 
 	create_lighting_overlays()
 
 /datum/controller/process/lighting/doWork()
+	// Counters
```

Comment:
I wasn't gonna say it but 99% of the comments in these files are pointless and you're just wasting space